tasks/main.yml

```diff
@@ -31,16 +31,23 @@
       tags:
         - molecule-idempotence-notest
 
+    # The lastest cryptsetup-initramfs is needed to fix upgrade
     - name: Reinstall cryptsetup-initramfs so upgrade doesn't fail
       apt:
         name: 'cryptsetup-initramfs'
+        # ansible-lint generates a warning that "package installs should
+        # not use latest" here, but this is one place where we want to use
+        # it.
+        state: latest  # noqa 403
 
     # Upgrade with command module instead of apt ansible module
-    #
+    # and ignore configuration file changes
```

Comment:
Why do you have to ignore configuration file changes here?  I think a comment explaining that would be a good idea.

Reply:
Addressed in commit a130bc5